pyvrp/tests/test_Population.py

```diff
@@ -257,12 +277,13 @@ def test_elite_individuals_are_not_purged(nb_elite: int):
 
 
 def test_binary_tournament_ranks_by_fitness():
-    data = read("data/RC208.txt", "solomon", "dimacs")
+    data = read("data/RC208LessVehicles.txt", "solomon", "dimacs")
```

Comment:
Why does this need a different (new) instance?

Reply:
Because the test assumes that the randomly generated individuals are all infeasible, which was not the case when I ran the test with the original file.

Reply:
But we can just filter the feasible solutions away before passing `init` to `pop`, right?